### PR TITLE
Add an appModule for zoom, fixes #7754

### LINKED ISSUE #7754: Zoom Does Not work As Expected with NVDA
### Steps to reproduce:

1. Go to this link and install the zoom client: https://zoom.us/download
1. Install the program.
1. When you launch the program, tab to either "start a meeting with video", or "start a meeting without video". Press enter on either of these buttons.
1. Press alt+a to mute or un-mute your audio. Press alt+v to turn your video on or off. 
1. Press alt+h to open the chat panel.You are immediately put into the edit field to type text. Type a few words. use your arrow keys to move around the text, mostly the left and right arrow.  

### Expected behavior:

For step 4 above, you should hear the fact that audio is either muted or un-muted, or that video is started or stopped. For step 5 above, you should be able to navigate the text with arrow keys after typing it. It should also be noted that Zoom gives notifications of everything that goes on in the room. For example, when recording is stopped or started, what someone types in the chat panel, when screen sharing is started or stopped, and so on.

### Actual behavior:

When I do the above steps, I get no notification of anything having occurred. I receive nothing telling me that an action has been taken. I cannot see text that I type into any edit field within Zoom by using arrow keys. I can, however, hear the text when I tab away from and then come back to the edit field.  

System configuration:
Dual-core 2.16 ghz Intel Celeron, 4 gb ram, 500 gb hard drive. Video card is on-board Intel video.
NVDA version:
NVDA next-14601,e62a168e 
> NVDA Installed or portable:
Installed

 Other information:
> Example: Running in a VM

Windows version:
> Windows 6.3 build 9600

Name and version of other software in use when reproducing the issue:

The absolutely latest version of Zoom
### Other questions:

Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
> No 
Please list them and the result
Roommate uses version 2017.3, and he has same issues that I described above.

### PR DESCRIPTION
**Pull request by derekriemer**
### Link to issue number:
Fixes #7754

### Summary of the issue:
Make zoom's notifications via alerts accessible.

### Description of how this pull request fixes the issue:

In zoom, alerts are filtered out because zoom uses a window which is not a child of the foreground window for zoom.  This pull request requests events for the  window class the notifications use, even in the background.

### Testing performed:

Tested to see if alerts worked while in zoom.

### Known issues with pull request:

None

### Change log entry:

Section: Bug fixes

### CODE REVIEW

**Comment by josephsl** [comment]
Hi,

Hmmm, if what's new entry is added, this could set a precedent for others. @LeonardDer and others, any comments on this?

Thanks.b

**Comment by PratikP1** [comment]
*   Hmmm, if what's new entry is added, this could set a precedent for others
I’m not sure what this means. Do you not want people to know that such a module was added?

**Comment by PratikP1** [comment]
Ah. Thank you for that explanation.

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -2394,4 +2395,3 @@ Major highlights of this release include support for 64 bit editions of Windows;
 - NVDA now asks if it should save configuration and restart if the user has just changed the language in the User Interface Settings Dialog. NVDA must be restarted for the language change to fully take effect.
 - If a synthesizer can not be loaded, when choosing it from the synthesizer dialog, a message box alerts the user to the fact.
 - When loading a synthesizer for the first time, NVDA lets the synthesizer choose the most suitable voice, rate and pitch parameters, rather than forcing it to defaults it thinks are ok. This fixes a problem where Eloquence and Viavoice sapi4 synths start speaking way too fast for the first time.
-
```
Comment (leonardder) [inline user_docs/en/changes.t2t]: It looks like this line got accidentally removed

**Review by LeonarddeR — DISMISSED** [review]
Note that I honestly haven't used this software, however I assume you've tested your implementation.

**Comment by josephsl** [comment]
Hi, one more thing: could you mind adding copyright headers please? Thanks.

File: user_docs/en/changes.t2t (hunk 1 above)
Comment (feerrenrut) [reply user_docs/en/changes.t2t]: Was this intentional?

**Review by feerrenrut — COMMENTED** [review]
Also, I will have to rely on your testing. Can you get the original reporter to confirm this addresses the issue for them with a try/pr build?

**Comment by derekriemer** [comment]
Copyright headers are there. Yes, will do.

**Comment by derekriemer** [comment]
I rebased this on master.

**Comment by derekriemer** [comment]
I didn't mean to remove that line, sorry, I was messing with the online github file editor. This poses the question, why is there two lines at the end of that file though?

**Comment by derekriemer** [comment]
@feerrenrut  multiple reports on the parent ticket now confirm my try build works as expected.

**Review by feerrenrut — DISMISSED** [review]

**Review by feerrenrut — APPROVED** [review]

**Pull request merged**